great-idea-website/css/index.css

```diff
@@ -63,4 +63,124 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+body{
+	margin: 40px;
+}
+.navbar{
+	text-align: left;
+	padding-bottom: 60px;
+}
+.navbar a{
+	display: inline-block;
+	padding-left: 40px;
+	padding-right: 60px;
+	justify-content: space-evenly;
+}
+.logo{
+	display: inline-block;
+	float: top;
+	padding-left: 90px;
+
+}
+.content_1{
+
+	width: 100%;
+}
+.innovate{
+	display: inline-block;
+	float: left;
+	margin-top: 5%;
+	margin-right: -5%;
+	margin-left: 7%;
+	width: 50%;
+	text-align: center;
+	font-size: 65px;
+}
+.content_1_box{
+	display: inline-block;
+	padding-top: 10px;
+	margin-left: -12%;
+	float: left;
+	width: 50%;
+	text-align: center;
+	font-size: 70px;
+}
+#started{
+	padding-top: 10px;
+	padding-right: 50px;
+	padding-bottom: 10px;
+	padding-left: 50px;
+	background-color: whitesmoke;
+}
+p{
+	font-size: inherit;
+}
+.overview{
+	width: 100%;
+	margin: 1%;
+	
+}
+#features{
+	display: inline-block;
+    min-height: 100px;
+    margin: 1%;
+	width: 47%;
+	/* margin-left: auto;
+	margin-right: auto;
+	 */
+	 margin-bottom: 30px;
+}
+#about{
+	display: inline-block;
+    min-height: 100px;
+    margin: 1%;
+	width: 47%;
+	margin-left: auto;
+	margin-right: auto;
+	margin-bottom: 30px;
+	 
+}
+.center{
+	display: block;
+	width: 1500px;
+	height: 200px;
+	margin-bottom: 30px;
+}
+.middle-img{
+	position: relative;
+	max-width: 100%;
+	max-height: auto;
+	
+}
+#services{
+	 display: inline-block;
+	 float:left;   
+     width:33.33%; 
+	 height:250px;
+	 
+}
+#product{
+	display: inline-block;
+	float:left; 
+	width:33.33%; 
+    height:250px;
+
+}
+#vision{
+	 float:left;  
+     width:33.33%; 
+     height:250px;
```

Comment:
So far everything looks good. I'll have to see how it turned out tomorrow. The only critique I have is a small one. Make sure your code is neat. Have everything lined up evenly and uniformed. Might seem like a petty thing to critique right now but once we start getting into LESS and even JSX you will see how important it is to have neat code.